Makefile

```diff
@@ -250,8 +250,6 @@ pytest_macOS_for_GitHub:
 	--randomly-dont-reset-seed \
 	${PYTEST_OPTIONS}
 
-# Warning: collecting tests can trigger the parameter_search flaky test
-# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3812
 .PHONY: check_currentl_flaky_tests # Print the current list of known flaky tests
```

Comment:
just realized there's a small typo here `check_currentl_flaky_tests` should be `check_current_flaky_tests`